src/System.IO.IsolatedStorage/tests/System/IO/IsolatedStorage/ContainsUnknownFilesTests.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Reflection;
+using Xunit;
+
+namespace System.IO.IsolatedStorage
+{
+    public class ContainsUnknownFilesTests : IsoStorageTest
+    {
+        private static MethodInfo s_containsUnknownFilesMethod;
+
+        static ContainsUnknownFilesTests()
+        {
+            s_containsUnknownFilesMethod = typeof(IsolatedStorageFile).GetMethod("ContainsUnknownFiles", BindingFlags.NonPublic | BindingFlags.Instance);
```

Comment:
I need to make a pass through and turn on the desktop tests (which I consider part of this overall workitem). Some tests apply, some don't. I've tried (in cases like this) to use the same sigs so I can reuse the same tests.